lms/migrations/versions/b11713314988_email_digest_enable_01.py

```diff
@@ -26,21 +26,24 @@ def enable_email_digest(conn, limit=500):
                 ELSE jsonb_set(settings, '{hypothesis,instructor_email_digests_enabled}', 'true')
             END
         FROM (
-            SELECT application_instance_id
-            FROM assignment_membership
-            JOIN "user" ON "user".id = assignment_membership.user_id
-            JOIN lti_role ON lti_role.id = lti_role_id
-            JOIN application_instances ON application_instance_id = application_instances.id
-            WHERE
-                -- Teachers
-                type ='instructor' AND scope = 'course'
-                -- AIs with some recent activity
-                AND last_launched >= '2023-01-01'
-                -- With the feature currently disabled
-                AND COALESCE(settings->'hypothesis'->>'instructor_email_digests_enabled', 'false') = 'false'
-            GROUP BY application_instance_id, "user".id
-            -- Least used first
-            ORDER BY COUNT("user".id) ASC
+            SELECT distinct application_instance_id
```

Comment:
Previus batch didn't update 500 rows as the query was returning duplicates :facepalm: